### PR TITLE
Add TeraSky to the TCE Adopters file

### PR DESCRIPTION
**Pull request by vrabbi**
## What this PR does / why we need it

Add TeraSky to the TCE ADOPTERS.md
## Details for the Release Notes (PLEASE PROVIDE)

```release-note
NONE
```

## Which issue(s) this PR fixes

Fixes: #

### CODE REVIEW

**Review by stmcginnis — DISMISSED** [review]
Aweseome!

**Comment by pnbrown** [comment]
Thanks for submitting this Scott.

Could we get this more like the Carvel format of logo at the top and description at the bottom?
https://github.com/vmware-tanzu/carvel/blob/develop/ADOPTERS.md

**Comment by vrabbi** [comment]
there seems to be an issue with the URL Linter accepting the Certificate of the Link to the terasky website. also @pnbrown, i made the changes to be like the Carvel ADOPTERS.md but now the mdlinter is complaining as it doesnt allow embedded HTML so not sure what to do about this

**Comment by jpmcb** [comment]
> There seems to be an issue with the URL Linter accepting the Certificate of the Link to the terasky website

We are looking at that now. Shouldn't be a blocker, we are considering setting the skip insecure verification since we the URL linter shouldn't really care about if the cert is valid, just that it at least goes somewhere. Stay tuned!

File: ADOPTERS.md [hunk 1]
```diff
@@ -4,6 +4,12 @@ If you're using Tanzu Community Edition and want to add your organization to thi
 
 ## Organizations using Tanzu Community Edition
 
+<a href="https://beam.lu/" border="0" target="_blank"><img alt="Beam" src="logos/terasky.png" height="50"></a>
```
Comment (stmcginnis) [inline ADOPTERS.md:7]: You can change this to:

```md
![terasky logo](logos/terasky.png)
```

Or to have it be a link (probably not to beam.lu though), I think you can do:

```md
[![terasky logo](logos/terasky.png)](https://terasky.com/)

Comment (vrabbi) [reply ADOPTERS.md]: awesome just pushed the change. thanks for the help!

**Pull request merged**